FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Models/IApiDataClient.cs

```diff
@@ -5,27 +5,26 @@
 
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Health.Fhir.Synapse.DataClient.Models.FhirApiOption;
 
-namespace Microsoft.Health.Fhir.Synapse.DataClient
+namespace Microsoft.Health.Fhir.Synapse.DataClient.Models
```

Comment:
Nit: we can put common interfaces at the root namespace of the project instead of `Models`, just like `IDataClient`, `ISchemaManager`, etc., as they are commonly used across projects.

Reply:
Sure, updated.